### PR TITLE
Refactor commands

### PR DESCRIPTION
**Pull request by eqrx**

### CODE REVIEW

File: internal/packages/packageexport/image_test.go [hunk 1]
```diff
@@ -29,50 +25,16 @@ func TestImage(t *testing.T) {
...
-func (w *writer) WriteHeader(statusCode int) { w.resp.StatusCode = statusCode }
-
-type tripper struct {
-	handler http.Handler
-}
-
-func (t tripper) RoundTrip(req *http.Request) (*http.Response, error) {
-	if req.Body == nil {
-		req.Body = io.NopCloser(&bytes.Buffer{})
+func Push(ctx context.Context, references []string, image v1.Image, opts ...crane.Option) error {
```
Comment (garrettthomaskth) [inline internal/packages/packageexport/image_test.go:28]: Is `Push` used somewhere?

File: cmd/kubectl-package/export/push_test.go [hunk 2]
```diff
@@ -1,65 +0,0 @@
...
-		req.Body = io.NopCloser(&bytes.Buffer{})
-	}
-	resp := &http.Response{Status: "ok", StatusCode: http.StatusOK, Header: http.Header{}, Request: req}
-	w := &writer{resp}
-	t.handler.ServeHTTP(w, req)
-
-	return resp, nil
-}
-
-func TestPush(t *testing.T) {
```
Comment (garrettthomaskth) [inline cmd/kubectl-package/export/push_test.go:49]: Can we save this TestPush test and put it in `internal/packages/packageexport/image_test.go`

Comment (eqrx) [reply cmd/kubectl-package/export/push_test.go:49]: Oops, yeah I messed that one up.

File: internal/packages/packageexport/image_test.go (hunk 1 above)
Comment (eqrx) [reply internal/packages/packageexport/image_test.go:28]: Nope, copied the wrong method xD

**Pull request merged**